### ISSUE TITLE
Treatment of bias adjustment - historical and projection period in MSE

### ISSUE THREAD

**Issue by aaronmberger-nwfsc** [pacific-hake/pacifichakemse#4]
Just starting a thought thread here on the subject of the "weighted" bias adjustment feature in stock synthesis (sensu [Methot and Taylor](https://www.researchgate.net/profile/Richard_Methot/publication/237176041_Adjusting_for_bias_due_to_variability_of_estimated_recruitments_in_fishery_assessment_models/links/00b7d5314cf24a6bb9000000/Adjusting-for-bias-due-to-variability-of-estimated-recruitments-in-fishery-assessment-models.pdf)).  I'm going to download some thoughts I have here with hopes to spark some discussion.  

General issue wrt Pacific Hake MSE workflow: 
OM contains two “parts” – a pre-projection period which is made to match the historical period as estimated using the 2018 (or year y) hake stock assessment model; and a projection period which is based on parameters from the historical period as a whole and/or parameters averaged over the most recent years during the historical period.  The pre-projection period OM was conditioned to be reasonable given information about the stock from the operation stock assessment.  During projections, recruitment is drawn using the historical period estimated stock-recruitment curve with future recruitment deviates drawn from a lognormal distribution with sigmaR: log recruitment deviation ~ N(0,sigmaR).  Note that there is the usual bias adjustment (-sigma_squared/2)  that pertains to ensuring scales are maintained when moving between lognormal and nominal.  There is the addition of a year-specific "weighting" factor bias adjustment specification embedded in Stock Synthesis (sensu Methot and Taylor) to account for levels of recruitment information when making the bias adjustment.

The reason the model has thus far included a "weighted" bias adjustment for future projections is because you want the S/R relationship to be maintained from the historical period on through the projection period as it maintains equilibrium (e.g., R0) baseline conditions (e.g., B0) for comparing reference points.  This was particularly important for the OM conditioning step.

Current approach:
Tune the OM bias adjustment scalar so that equilibrium stock size (and recruitment) are maintained between the operation stock assessments estimates and the MSE EM as well as during the projection period for the MSE OM.  This general approach has brought up questions on why an OM when need to be "adjusted", as it pertains to the "truth".  The answer lies with the preservation of B0 (and R0) when transitioning from the historical period to the projection period so that performance metrics related to depletion (or reference points in general) are representative of values used in the current management system.
 
[side worry: once we add alternative OM governing population dynamic starting conditions (e.g., steepness estimates) this will change how a static bias adjustment affects unfished conditions for example- will need to think about this]

Initial thoughts on an alternative approach: 

Step 1 – condition the model so that the median starting conditions for the projection period are reasonable. [This has already been done].

Step 2 – project the median (point estimate or HPD estimates) into the future with no fishing for many years (~100 years) to get “unfished conditions” and equilibrium values while using the basic lognormal bias adjustment of Recruitment = S/R curve * exp(rec dev – sigmaR^2/2)  -- (i.e., SS internal bias adjust ‘weight’ = 1.0 – because OM futures are perfectly known). [the question still arises of what to do for alternative starting conditions?  Do this each run or just use the point estimates (HPD estimates) as an average baseline.]

Step 3 – run the MSE using alternative starting conditions and from hence forth ignore the historical period and the EM uses a bias adjust similar to what is always used (with ramp down near end of time period for 2-3 years).  This will cause bias relative to the OM, but that is just the “truth” of it when we do assessments with no recent recruitment information.

Step 4 – performance metrics that are based on reference points using unfished conditions (e.g., B0) will be based on equilibrium conditions estimated in step 2.  Note that B0 and R0 here will likely be a bit different from the operation stock assessment model, but that should not really matter when comparing scenarios and should only slightly differ by a small constant.  If it became a concern, we could compare performance metrics when using the operational assessment levels of B0 versus a B0 estimated via equilibrium seeking projections with no fishing. 

Sorry for the long winded and perhaps hard to follow thinking aloud brain dump.  What am I missing with this bias adjustment stuff?

**Comment by iantaylor-NOAA** [comment]
@aaronmberger-nwfsc, I like this idea.
This is a lot like a dynamic B0 (what would the unfished biomass be under the observed recruitments), only it could be an average across many years and/or iterations. I'm not sure that the bias you suggest will be present in the EM in step 3 is a foregone conclusion. There will definitely be variability in the EM estimates due to lack of information about recent recruitment, but I'm not sure they will be biased for sure.

**Comment by kristinmarshall-NOAA** [comment]
I like this idea too.  

The way the current approach is described aligns with my understanding.  One additional piece that still puzzles me about the current approach is why the assessment uses a weight of 0.87, but after tuning,  OM needs a value of 0.5 to get to the same B0 as the assessment. Why this value?  Do we need to understand more about this difference in scaling, or is it enough to say this is what it is? 

It's less important to dig into if we move to a new approach though...
